setup.cfg

```diff
@@ -25,6 +25,7 @@ testpaths = tests/unittest/
 
 filterwarnings =
     error
+    ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning:falcon
```

Comment:
What's kicking up this error? We should add a comment above it just like we do for the below one.